dask/dataframe/io/parquet/core.py

```diff
@@ -184,15 +185,35 @@ def read_parquet(
     read_from_paths : bool, default None
         Only used by ``ArrowDatasetEngine`` when ``filters`` are specified.
         Determines whether the engine should avoid inserting large pyarrow
-        (``ParquetFileFragment``) objects in the task graph. If this option
-        is True (default), ``read_partition`` will need to regenerate the
-        appropriate fragment object from the path and row-group IDs.  This will
-        reduce the size of the task graph, but will add minor overhead to
-        ``read_partition``.
-    chunksize : int, str
-        The target task partition size.  If set, consecutive row-groups
-        from the same file will be aggregated into the same output
-        partition until the aggregate size reaches this value.
+        (``ParquetFileFragment``) objects in the task graph.  If this option
+        is True, ``read_partition`` will need to regenerate the appropriate
+        fragment object from the path and row-group IDs.  This will reduce the
+        size of the task graph, but will add minor overhead to ``read_partition``.
+        By default (None), ``ArrowDatasetEngine`` will set this option to
+        ``False`` when there are filters.
+    chunksize : int, str, tuple, default None
+        The target output partition size. If specified, adjacent row-groups
```

Comment:
What size does this refer to? Compressed parquet? uncompressed arrow? deserialized pandas? If there is an easy answer to this question, can you please put this in the doc string?

Reply:
Good question.  We are using the ``total_byte_size`` statistic, which corresponds to the uncompressed parquet-storage size.  I updated the docstring.